### PR TITLE
[minor] Add support for OCP IPI on GCP

### PR DESCRIPTION
**Pull request by vskhabani**
Added changes related to GCP IPI support in ocp_provision and ocp_deprovision role.

OpenShift Container Platform requires a Google Cloud Platform (GCP) service account that provides authentication and authorization to access data in the Google APIs. Service Account file will be a json file which can be downloaded from GCP console.
By default openshift-installer checks service account into the env variable `GOOGLE_APPLICATION_CREDENTIALS` .
`gcp_service_account_file`  variable takes value present in the env `GOOGLE_APPLICATION_CREDENTIALS`

In ipi install-config.yaml.j2 added condition to use project_id if ipi_platform is gcp which is required specifically for the GCP.

Specific Openshift documentation: https://docs.openshift.com/container-platform/4.7/installing/installing_gcp/installing-gcp-customizations.html

Used this role to provision OpenShift Cluster on GCP account.

### CODE REVIEW

**Review by whitfiea — COMMENTED** [review]
@vskhabani why is the env var called `GOOGLE_APPLICATION_CREDENTIALS` and not `GCP_SERVICE_ACCOUNT_FILE`? How is that file then used as the only changes seem to be to check the file exists but doesn't get used anywhere?

**Review by whitfiea — COMMENTED** [review]
@vskhabani so I did some googling the it seems that GOOGLE_APPLICATION_CREDENTIALS is the expected env var in a GCP environment, and we are just assigning it to `gcp_service_account_file` to do a check only in our ansible?

**Review by whitfiea — APPROVED** [review]

**Review by durera — CHANGES_REQUESTED** [review]
Please add an update to the changelogs for 11.5 (which this version will be), and this is good to merge:

here: https://github.com/ibm-mas/ansible-devops/tree/ipi-gcp/ibm/mas_devops#change-log
and here: https://github.com/ibm-mas/ansible-devops/blob/ipi-gcp/docs/changes.md#changes

**Comment by vskhabani** [comment]
@durera Updated the changelogs for 11.5

File: docs/changes.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ## Changes
 Note that links to pull requests prior to public release of the code (4.0) direct to IBM GitHub Enterprise, and will only be accessible to IBM employees.
-
+- `11.5` Multiple Updates:
```
Comment (durera) [inline docs/changes.md:3]: Don't need to seperate out provision and deprovision, one entry per PR basically .... should just be:
```
- `11.5` Add support for OCP IPI on GCP ([#483](https://github.com/ibm-mas/ansible-devops/pull/483))
```

**Review by durera — CHANGES_REQUESTED** [review]

File: docs/changes.md (hunk 1 above)
Comment (vskhabani) [reply docs/changes.md:3]: Updated.

**Pull request merged**